client/components/site-selector/index.jsx

```diff
@@ -264,13 +264,12 @@ const SiteSelector = React.createClass( {
 	},
 
 	render() {
+		const hiddenSitesCount = this.props.siteCount - this.props.visibleSiteCount;
 		const selectorClass = classNames( 'site-selector', 'sites-list', {
-			'is-large': this.props.siteCount > 6,
+			'is-large': this.props.siteCount > 6 || hiddenSitesCount > 0,
 			'is-single': this.props.visibleSiteCount === 1
```

Comment:
>The is-single?

[Yes](https://github.com/Automattic/wp-calypso/search?utf8=%E2%9C%93&q=.is-single).